samples/kotlin-android-moshi-codegen/build.gradle.kts

```diff
@@ -44,14 +44,14 @@ dependencies {
 }
 
 generateSwagger {
-    platform = "kotlin"
-    packageName = "com.yelp.codegen.samples"
-    specName = "sample_specs"
-    specVersion = "1.0.0"
-    inputFile = file("../sample_specs.json")
-    outputDir = file("./src/main/java/")
+    platform.set("kotlin")
+    packageName.set("com.yelp.codegen.samples")
+    specName.set("sample_specs")
+    specVersion.set("1.0.0")
+    inputFile.set(file("../sample_specs.json"))
+    outputDir.set(file("./src/main/java/"))
     features {
-        headersToRemove = arrayOf("Accept-Language")
+        headersToRemove.add("Accept-Language")
     }
```

Comment:
Moving to lazy configuration changes the plugin API. This certainly requires a major version, some warnings or something else.

Reply:
Yup I've added this PR to the `2.0.0` milestone